<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ingenieria</class>
 <widget class="QMainWindow" name="Ingenieria">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1864</width>
    <height>1225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingenieria</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Color de fondo de la ventana */
QMainWindow {
    background-color: #0b0b0b;
}

/* Estilo para la barra de menús */
QMenuBar {
    background-color: #1a1a1a;
    color: #ffffff; /* Letras blancas */
    border-bottom: 1px solid #333333;
}

/* Estilo para los ítems del menú al pasar el mouse */
QMenuBar::item:selected {
    background-color: #00ccff; /* Color neón de tu diseño */
    color: black;
}

/* Estilo para los menús desplegables */
QMenu {
    background-color: #1a1a1a;
    color: white;
    border: 1px solid #333333;
}

QMenu::item:selected {
    background-color: #00ccff;
    color: black;
}</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-21</x>
      <y>-91</y>
      <width>1911</width>
      <height>1281</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="page_3">
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>69</y>
           <width>1871</width>
           <height>1201</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>-11</x>
               <y>9</y>
               <width>2081</width>
               <height>1181</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QWebEngineView" name="cargar_dashboard">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="url">
                 <url>
                  <string>about:blank</string>
                 </url>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1864</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuDashboard">
    <property name="title">
     <string>Dashboard</string>
    </property>
    <addaction name="actionDashboard"/>
   </widget>
   <widget class="QMenu" name="menuGesti_n_De_Requisito">
    <property name="title">
     <string>Gestión De Requisito</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDise_o_Y_Modelado">
    <property name="title">
     <string>Diseño Y Modelado</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLista_De_Materiales">
    <property name="title">
     <string>Lista De Materiales</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuControl_De_Cambios">
    <property name="title">
     <string>Control De Cambios</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAseguramiento_De_Calidad">
    <property name="title">
     <string>Aseguramiento De Calidad</string>
    </property>
   </widget>
   <addaction name="menuDashboard"/>
   <addaction name="menuGesti_n_De_Requisito"/>
   <addaction name="menuDise_o_Y_Modelado"/>
   <addaction name="menuLista_De_Materiales"/>
   <addaction name="menuControl_De_Cambios"/>
   <addaction name="menuAseguramiento_De_Calidad"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDashboard">
   <property name="text">
    <string>Grafíca</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
